src/index.js

```diff
@@ -163,18 +166,25 @@ export async function findPath (params) {
  * @param {URI} account
  * @returns {Promise<Account>}
  */
-async function getAccount (account) {
-  const res = await request.get(account)
-  if (res.statusCode !== 200) {
-    throw new Error('Unable to identify ledger from account: ' + account)
-  }
-  return res.body
+function getAccount (account) {
+  return co(function * () {
+    const res = yield request.get(account)
+    if (res.statusCode !== 200) {
+      throw new Error('Unable to identify ledger from account: ' + account)
+    }
+    return res.body
+  })
 }
 
 /**
  * @param {URI} account
  * @returns {Promise<URI>}
  */
-async function getAccountLedger (account) {
-  return (await getAccount(account)).ledger
+function getAccountLedger (account) {
+  return getAccount(account).then(account => account.ledger)
 }
+
+module.exports = sendPayment
+exports.default = sendPayment
```

Comment:
This doesn't do anything -- you just set `module.exports` to something else, but `exports` doesn't automatically change, so `default`, `executePayment`, and `findPath` aren't being exported.